src/StackTask/Program.cs

```diff
@@ -1,2 +1,78 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using System;
+
+namespace StackTask
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            var stack = new STACK<char>();
+            stack.Push('r');
+            stack.Push('b');
+            stack.Push('q');
+            stack.Pop();
+            stack.Push('h');
+            // stack.Clear();
+            stack.Print();
+        }
+    }
+}
+
+public class STACK<T> 
+{
+    public class Node<T>
+    {
+        public Node<T> next = null;
+        public T data;
```

Comment:
Same comment as above, should be in pascalcase `Data`.

Reply:
Done